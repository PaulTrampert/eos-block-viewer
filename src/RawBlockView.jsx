import React from 'react';
import PropTypes from 'prop-types';

const RawBlockView = ({block}) => (
  <pre>
    {JSON.stringify(block, null, 2)}
  </pre>
);

RawBlockView.propTypes = {
  block: PropTypes.object
};

export default RawBlockView;
