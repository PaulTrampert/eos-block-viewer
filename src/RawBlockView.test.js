import React from 'react';
import {shallow} from 'enzyme';
import RawBlockView from './RawBlockView';

describe("RawBlockView", () => {
  let subject;
  let block;

  beforeEach(() => {
    block = {

    };
    subject = shallow(<RawBlockView block={block} />);
  });

  it('renders the stringified block within <pre> tags', () => {
    expect(subject).toMatchSnapshot();
  });
});