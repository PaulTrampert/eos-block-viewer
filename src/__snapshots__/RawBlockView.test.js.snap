// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RawBlockView renders the stringified block within <pre> tags 1`] = `
<pre>
  {}
</pre>
`;
